### ISSUE TITLE
DCenterArranger doesnt set Children[i].Last correctly

### ISSUE THREAD

**Issue by eugeneloza** [eugeneloza/decoherence#457]
`DCenterArranger` is not animating as it should.
The reason is that it doesn't change the start animation position of `Children` when initializing. And practically it shouldn't, as they must be created in a specific way.
While this is expected behavior, I don't like it :)

While not important for most arrangers, it's critical that `DPortrait` (being a `DCenterArranger` child) is animated in a homogeneous way from the very first time. Preferably, if this is achieved automatically, not by specifying the _correct_ start values in `SpawnChildren`, because they can change easily, thou still must be arranged against center.

So, basically we should:
Arrange `Last` (or, rather `Current`) of the `Children` against `Self.Current`. <----- for some reason this fails
Arrange `Next` (as it is now)
